classes/class-paysoncheckout-for-woocommerce-callbacks.php

```diff
@@ -139,11 +139,6 @@ public function pco_check_for_order_callback( $payment_id ) {
 			} else {
 				$result = $order_confirmation->confirm_payson_order( $order->get_id() );
 			}
-
-			if ( $result ) {
-				$order->add_order_note( __( 'Order confirmed on a callback from Payson.', 'woocommerce-gateway-payson' ) );
-				PaysonCheckout_For_WooCommerce_Logger::log( 'Order confirmed on a callback from Payson. Payment id: ' . $payment_id . ' Order id: ' . $order->get_id() );
-			}
```

Comment:
Im not sure we should remove the callback specific logs, rather add a log during the actual confirmation step as well. Try placing a order normally and see what shows when no callback is needed to process the order, and we might just need to add a log entry during the actual confirmation step instead.